app/src/main/AndroidManifest.xml

```diff
@@ -19,15 +19,17 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".activities.FocusParentActivity"></activity>
         <activity android:name=".activities.MeditationHome"></activity>
+        <activity
+            android:name=".activities.FocusParentActivity"
+            android:noHistory="true" />
         <activity
             android:name=".activities.TestModeActivity"
             android:label="@string/title_activity_test_mode"
             android:parentActivityName=".main.NeuroLab" />
         <activity
             android:name=".activities.ProgramModeActivity"
-            android:parentActivityName=".main.NeuroLab" />
+            android:noHistory="true" />
```

Comment:
I wonder why we are using such setting?

Reply:
This served as a parent Activity for all the program mode fragments but after inducing the FocusModeParent Activity which serves parent to the ProgramModeActivity itself created exception.
Simply put, otherwise it created lifecycle stack problems.